src/app/helptext/storage/snapshots/snapshots.ts

```diff
@@ -24,24 +24,42 @@ snapshot_add_recursive_tooltip: T('Set to include child datasets of the \
 
 // storage/snapshots/snapshot-clone
 snapshot_clone_name_placeholder: T('Name'),
-snapshot_clone_name_tooltip: T('Name of the new dataset created from the cloned snapshot.'),
+snapshot_clone_name_tooltip: T('Name of the new dataset created from the \
+ cloned snapshot.'),
 snapshot_clone_name_validation : [ Validators.required ],
 
 label_clone: T('Clone to New Dataset'),
 label_delete: T('Delete'),
 label_rollback: T('Rollback'),
 
 rollback_snapshot_placeholder: T('Snapshot'),
-rollback_recursive_radio_placeholder: T('Rollback options'),
-rollback_recursive_radio_tooltip: T('Select an option for rolling back.'),
-rollback_dataset_placeholder: T('This dataset'),
-rollback_dataset_tooltiP: T('Setting this will rollback only the current dataset, leaving any child datasets or clones unchanged.'),
-rollback_recursive_placeholder: T('This dataset and child datasets'),
-rollback_recursive_tooltip: T('Setting this will rollback child datasets as well as the current dataset, destroying any newer snapshots on the children as well as this one.'),
-rollback_recursive_clones_placeholder: T('This dataset, child datasets and clones'),
-rollback_recursive_clones_tooltip: T('Setting this will rollback child datasets and clones as well as the current dataset, destroying any newer snapshots on the children as well as this one.'),
-rollback_warning: T("<b>WARNING:</b> Rolling back to this snapshot will permanently delete later snapshots of this dataset.\
- Do not roll back until all desired snapshots have been backed up!"),
-rollback_title: ('Rollback Snapshot'),
+
+rollback_recursive_radio_placeholder: T('Stop Rollback if Snapshots Exist:'),
+rollback_recursive_radio_tooltip: T('Choose a safety level for the \
+ rollback action. The rollback is canceled when the safety check finds \
+ additional snapshots that are directly related to the dataset being rolled back.'),
+
+rollback_dataset_placeholder: T('Newer Intermediate, Child, and Clone'),
+rollback_dataset_tooltip: T('Stops the rollback when the safety check \
+ finds any related intermediate, child dataset, or clone snapshots that \
+ are newer than the rollback snapshot.'),
+
+rollback_recursive_placeholder: T('Newer Clone'),
+rollback_recursive_tooltip: T('Stops the rollback when the safety \
+ check finds any related clone snapshots that are newer than the \
+ rollback snapshot.'),
+
+rollback_recursive_clones_placeholder: T('No Safety Check (WARNING)'),
```

Comment:
WARNING sounds like it will give a warning if there are problems.  I'd use CAUTION or maybe even CAUTION, POTENTIAL DATA LOSS.